examples/simple_system/ibex_simple_system.core

```diff
@@ -87,4 +87,5 @@ targets:
           # XXX: Cleanup all warnings and remove this option
           # (or make it more fine-grained at least)
           - "-Wno-fatal"
+          - "--unroll-count 72"
```

Comment:
Can you add a comment why this is needed/helpful?

Reply:
Will do - it's a known verilator issue